Podfile.lock

```diff
@@ -604,7 +604,7 @@ DEPENDENCIES:
   - SVProgressHUD (= 2.2.5)
   - WordPress-Editor-iOS (~> 1.19.8)
   - WordPressAuthenticator (~> 2.0.0)
```

Comment:
Good question. I did create that authenticator new version and noticed it was prompted by changes used in WooCommerce.

I run `bundle exec pod install`, which, correctly, didn't update the version.

I run into a similar case in a past sprint, where I released 2.0.1 but didn't update here. My rationale, in both instances, is that the changes looked WooCommerce-specific.

Now that you call this out, I wonder whether I've been to cautious. The whole point of using SemVer is to leverage it to get upstream compatible updates. There might be new features in a release, but if the coding has been done properly, they should not affect the working of the app.

I'll open a PR against the release branch to update the library, and go from there.

Reply:
> I'll open a PR against the release branch to update the library, and go from there.

See https://github.com/wordpress-mobile/WordPress-iOS/pull/19163. I haven't made it ready for review yet because I wanted to check the auth screen with both 2.0.1 and 2.1.1.

Reply:
> I run into a similar case in a past sprint, where I released 2.0.1 but didn't update here. My rationale, in both instances, is that the changes looked WooCommerce-specific.
>
> Now that you call this out, I wonder whether I've been to cautious. The whole point of using SemVer is to leverage it to get upstream compatible updates. There might be new features in a release, but if the coding has been done properly, they should not affect the working of the app.

We have a similar problem with the FluxC library used in both WPAndroid and WCAndroid. Ideally if we were sure that the new versions of the library did follow SemVer, that wouldn't really be an issue. But in practice we don't maintain a CHANGELOG for those libs, nor do we have a tool to compare API changes like [`swift-package-diagnose-api-breaking-changes`](https://forums.swift.org/t/swift-package-diagnose-api-breaking-changes/54308). Which means that when we do a new version, it's hard for the release manager to know if the changes on the lib since last version were breaking changes or minor or patch bumps. Which means we're not guaranteed to follow SemVer in practice, and that a new version `2.1.0`, despite being a minor bump, could totally have broken something in WPiOS by accident compared to its `2.0.0`.

We should probably at some point maintain a `CHANGELOG.md` similar to [the one we use in the release-toolkit](https://github.com/wordpress-mobile/release-toolkit/blob/trunk/CHANGELOG.md), with explicit sections for what changes in the lib are breaking vs backwards-compatible additions vs bug fixes, so that we could garantee respecting SemVer when we pick a new version number, and thus avoid having to ask the question of whether it's safe for our client apps to update to a new release of the lib like here.

Reply:
> We should probably at some point maintain a `CHANGELOG.md`... 
💯